index.js

```diff
@@ -142,6 +147,8 @@ function RademacherHomePilot(log, config, api) {
                     body.meters.forEach(function(data) {
                         var uuid = UUIDGen.generate("did"+data.did);
                         var accessory = self.accessories[uuid];
+
+                    if ((config["DIDs"]).includes(data.did)) {
```

Comment:
add check for non-existing variable and empty list

Reply:
Inserted the DID as Option also changed the files around and made a check if the DIDs are set when want to use the feature.
Really not sure if its the best way how i inserted it .. i'm more comfortable with assembler, this is my first try to change a javascript in such a way with inserting new features.